manual/vcredist2008/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,27 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName         = 'vcredist2008'
+$softwareNamePattern = 'Microsoft Visual C++ 2008 Redistributable*'
+
+[array] $key = Get-UninstallRegistryKey $softwareNamePattern
+if ($key.Count -eq 1) {
+    $key | % {
+        $packageArgs = @{
+            packageName            = $packageName
+            silentArgs             = "/quiet /qn /norestart"
+            fileType               = 'msi'
+            validExitCodes         = @(0,3010) # http://msdn.microsoft.com/en-us/library/aa368542(VS.85).aspx
+            file                   = "$($_.UninstallString.Replace(' /x86=0', ''))"   #"C:\Program Files\OpenSSH\uninstall.exe" /x86=0
+        }
+        Uninstall-ChocolateyPackage @packageArgs
+    }
+}
+elseif ($key.Count -eq 0) {
+    Write-Warning "$packageName has already been uninstalled by other means."
+}
+elseif ($key.Count -gt 1) {
+    Write-Warning "${$key.Count} matches found!"
```

Comment:
As you see from the latest appveyor build, you can't use brackets, you need to wrap it in parenthesis.